.github/workflows/ci.yaml

```diff
@@ -105,12 +97,17 @@ jobs:
           juju model-defaults logging-config='<root>=INFO; unit=DEBUG'
           juju add-model test
 
-      - name: Terraform deploy
+      - name: Terraform deploy - simple deployments charm module
```

Comment:
I think its a good idea to also test the product modules. I could be wrong, but when you test the product modules you are implicitly testing the charm modules

[See how we do it in Mongo](https://github.com/canonical/mongodb-operator/blob/1c7a3f4acd7cd2f5c776d10ad63be6cf3c242fcf/.github/workflows/ci.yaml#L64-L111)